packages/theme/src/hooks/useTabs.tsx

```diff
@@ -3,24 +3,31 @@ import { TabContext, TabPanel, TabPanelClasses } from '@material-ui/lab'
 import { useEffect, useState } from 'react'
 import { ButtonGroupTabList, ButtonGroupTabListProps } from '../Components'
 
-export interface useTabsOptions {
+export interface useTabsOptions<T> {
```

Comment:
Please don't use `useTabs`. It has been removed in #3834 (maybe you can cherry-pick it)

Reply:
Maybe I can merge this into develop and remove it and commit in #3834 .What do you think?